core-models/src/main/scala/com/mesosphere/usi/core/models/template/SimpleRunTemplateFactory.scala

```diff
@@ -0,0 +1,87 @@
+package com.mesosphere.usi.core.models.template
+
+import com.mesosphere.usi.core.models.resources.ResourceRequirement
+import com.mesosphere.usi.core.models.{TaskName, TaskBuilder}
+import org.apache.mesos.v1.{Protos => Mesos}
+
+import scala.collection.JavaConverters._
+
+object SimpleRunTemplateFactory {
+
+  case class SimpleTaskInfoBuilder(
+      resourceRequirements: Seq[ResourceRequirement],
+      shellCommand: String,
+      role: String,
+      fetch: Seq[FetchUri] = Seq.empty,
+      dockerImageName: Option[String] = None)
+      extends TaskBuilder {
+
+    override def buildTask(
```

Comment:
I wonder if we could/should make this private to `[usi]`. What do you think?

Reply:
We cannot make it private; it's a publicly implementable trait method